scripts/release/util.js

```diff
@@ -49,12 +49,36 @@ const getElementList = async (directory) => {
     });
 };
 
+/*
+ * Return remove and replace unwanted text for md.
+ */
+const sanitize = (text) => {
```

Comment:
this could be helpful: `"some string".split('\n').map(e => e.trim()).join(' ')`

Reply:
I see. Let's add a comment describing why backticks must to be replaced.

> Not to mention that it can't handle content with multiple new lines.

This issue is yet to be resolved tho.